medical_appointment/data/medical_appointment_sequence.xml

```diff
@@ -1,12 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2015-TODAY
+	     LasLabs [https://laslabs.com]
+	     Komit [komit-consulting.com]
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
 
-<!--
-    Copyright (C) 2015-TODAY LasLabs [https://laslabs.com]
-    @author Dave Lasley <dave@laslabs.com>
-    @license AGPL-3
--->
-
-<openerp>
+<odoo>
```

Comment:
I did not know this worked.
I ll test, but if it works we should recommend an update here: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#format